src/content/en/fundamentals/payments/payment-apps-developer-guide/web-payment-apps.md

```diff
@@ -410,6 +410,25 @@ After the installation, the JIT-installable payment app is set as the supported
 payment method. The payment app can update it later as a previously-installed
 payment app.
 
+## Optional: Let Chrome skip-the-sheet
+
+Chrome can skip the Payment Request UI if certain conditions are met. By using
+this approach, you can let merchants put your own brand button and launch your
+payment app directly rather than launching the Payment Request UI and select the
+payment method your app supports.
+
+Following are the conditions to enable skip-the-sheet:
+
+1. Only one payment method identifier is specified in the `PaymentRequest`
+   constructor.
+2. No shipping address or contact info were requested in the `PaymentRequest`
```

Comment:
To be consistent, shouldn't a present tense he used instead of the past tense with "were"?

Reply:
Good catch. Will fix.